### DISCUSSION TITLE
How to change the Build region?

### DISCUSSION THREAD

**Discussion by ThomasSalty** [vercel/community#4247] [Help] [answered]
### Summary

I would like my build to happen in the "Frankfurt, Germany (West) - fra1" region because that's where my data source is. And this is what I set and saved in the Project settings for Function Region. I have also set "regions": ["fra1"] in my vercel.json just in case.

However, whenever I push changes to my feature branch and Vercel starts building and deploying the changes, then this is the first line in the Build logs:

"Running build in Washington, D.C., USA (East) – iad1 (Hive)"

How can I change the build region from Washington to Frankfurt?

### Steps to Reproduce

Basically, what I wrote in the summary.

Here are the screenshots:

![Vercel function region](https://github.com/vercel/community/assets/3471556/6d4e924e-25dc-475c-9445-40a34458b21d)

![Vercel build log](https://github.com/vercel/community/assets/3471556/dfaaeed3-3287-4a35-a9bf-52fc5e01e30a)

**Comment by ohyoxo** [comment]
I'm having the same problem, has it been solved yet?

**Comment by swarnava** [comment, marked as answer]
Hi,

This isn't possible at the moment and the only region you're able to change is the Serverless Function region and Edge Function regions. All build traffic will come from a US based regions and this can't be changed.

If you need a build to happen in a specific region, we'd suggest raising this as a feature request to allow our engineering team to better understand the need/public interest for this feature: https://vercel.com/guides/where-to-submit-feature-requests

Please let us know if you have any further questions or issues.
